backend/celery_worker/celery.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import absolute_import
+from celery import Celery
+import os
+from django.conf import settings
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "config.settings.local")
```

Comment:
Does this make celery use local settings even in prod environment?

Reply:
No, it works similarly to the same line in `manage.py`. It will only modify the variable it if it is not already been set in the container which it should have in the production environment.